templates/taxonomy-topic.php

```diff
@@ -1,23 +1,45 @@
-<?php get_header(); ?>
+<?php
+get_header();
+$title_classes = ( get_option( 'ucf_faq_include_athena_classes' ) ) ? " container mb-5" : "";
+?>
 
 <article>
-	<div class="container mb-5 ucf-faq-topic-list">
+	<div class="ucf-faq-topic-list<?php echo $title_classes; ?>">
 		<?php
 		$topic_description = term_description();
+
 		if ( ! empty( $topic_description ) ) {
-			echo '<div class="ucf-faq-topic-description mb-4">' . $topic_description . '</div>';
+			$topic_description_classes = "";
+
+			if( get_option( 'ucf_faq_include_athena_classes' ) ) {
+				$topic_description_classes = " mb-4";
+			}
+
+			echo '<div class="ucf-faq-topic-description'. $topic_description_classes . '">' . $topic_description . '</div>';
 		}
 
 		if ( have_posts() ) {
 			while ( have_posts() ) {
 				the_post();
+
+				$question_classes = "";
+				$question_attrs = "";
+				$answer_classes = "";
+				$answer_attrs = "";
+
+				if( get_option( 'ucf_faq_include_athena_classes' ) ) {
```

Comment:
This looks similar to https://github.com/UCF/UCF-FAQ-CPT/pull/5/files#diff-120b5ba5c344036409191b83b22c3110R33. It might be useful to separate this into a reusable function somehow.